components/prediction-tile/prediction-tile.tsx

```diff
@@ -52,23 +52,44 @@ export default function PredictionTile() {
     () => new Date(activePoll?.endTime || Date.now()).getTime() - Date.now(),
     [activePoll],
   );
-  const isPredictionExpired = useMemo(() => pollMsLeft <= 0, [pollMsLeft]);
+
+  const hasPredictionEnded = useMemo(() => pollMsLeft <= 0, [pollMsLeft]);
```

Comment:
looks like this is just a rename, but don't really understand why this is a memo